src/Controllers/QuestionInstanceController.php

```diff
@@ -107,17 +107,19 @@ public function getReviewPage(RequestInterface $request)
         $quizInstanceId = $this->session->get('quiz');
         $questionInstanceEntities = $this->questionInstanceService->getQuestionsInstance($quizInstanceId);
 
-        $textInstnaces = [];
+        $textInstances = [];
         foreach ($questionInstanceEntities as $key => $value) {
```

Comment:
Please extract the highlight functionality of answers in a separate class that can be used in all the places in which you need to highlight answers.